packages/editor/src/client/atoms/descriptionList.tsx

```diff
@@ -44,7 +45,16 @@ export function DescriptionListItem({label, children}: DescriptionListItemProps)
         style={{
           marginLeft: 20
         }}>
-        {children}
+        {children || (
+          <div
+            style={{
+              borderRadius: '8px',
+              padding: '6px',
+              backgroundColor: '#FFEBCD'
```

Comment:
Since we use rsuite.js i recommend to use those components. Maybe https://rsuitejs.com/components/message/ could be used here. If not we should at least reuse the colors for warnings and errors.

Reply:
I didn't use the rsuite component because the message was huge and didn't look good in the publish panel but I used its colors